images/wkdev_sdk/Containerfile

```diff
@@ -127,10 +127,12 @@ COPY /rootfs/usr/bin/podman-host /usr/bin/podman-host
 
 COPY /rootfs/etc/ccache.conf /etc/ccache.conf
 
-# Convenience symlink for clang tools, the VSCode extension doesn't find these by default.
-RUN for command in clang clang++ clangd clang-format clang-tidy lld lldb lldb-server lldb-vscode; do \
-        ln -s "/usr/bin/${command}-18" "/usr/local/bin/${command}"; \
-    done && ln -s "/usr/bin/lld-18" "/usr/local/bin/ld.lld";
+# Convenience symlinks for clang tools, the VSCode extension doesn't find these by default.
+# FIXME: Reduce code duplication with `wkdev-set-default-clang`.
```

Comment:
We should be able to invoke this script from here, no? Maybe at a later point in the script, but still..

Reply:
As long as you copy and remove in the same layer, there is no increase an image size -- I agree looks odd, but helps reducing code duplication...